util/version/version.go

```diff
@@ -201,17 +202,29 @@ func WithBuildTags() CompareOption {
 	}
 }
 
+func WithoutPreReleases() CompareOption {
+	return func(c *comparer) {
+		c.withoutPreReleases = true
+	}
+}
+
 // Compare 2 semver versions.
 // Defaults to doing the standard semver comparison when no options are specified.
 // The comparison logic can be modified by passing additional compare options.
 // Example: using the WithBuildTags() option modifies the compare logic to also
 // consider build tags when comparing versions.
 func Compare(a, b semver.Version, options ...CompareOption) int {
+
 	c := &comparer{}
```

Comment:
```suggestion
	c := &comparer{}
```
@SubhasmitaSw Last nit, golangci-lint doesn't like the leading newline

Reply:
@sbueringer  oh okay, might have missed it.